azure_functions_worker/main.py

```diff
@@ -26,8 +30,28 @@ def parse_args():
     return parser.parse_args()
 
 
+def setup_user_pkg_paths():
+    minor_version = sys.version_info[1]
+
+    home = Path.home()
+    pkgs_path = os.path.join(home, constants.PKGS_PATH)
+    venv_pkgs_path = os.path.join(home, constants.VENV_PKGS_PATH)
+
+    if minor_version == 6:
+        sys.path.insert(1, os.path.join(venv_pkgs_path, constants.PKGS_36))
+        sys.path.insert(1, os.path.join(pkgs_path, constants.PKGS_36))
+    elif minor_version == 7:
+        sys.path.insert(1, os.path.join(venv_pkgs_path, constants.PKGS_37))
+        sys.path.insert(1, os.path.join(pkgs_path, constants.PKGS_37))
```

Comment:
out of curiosity, why 1 and not 0 for these?

Reply:
We may want the currently directory path to have the top priority.

Reply:
I had wanted to keep the current directory path (i.e. `''`) as the first search path. This mimics the current behavior of the `PYTHONPATH` which is appended as the second entry in the `sys.path` after the current directory.